R/get_file_as_dataframe.R

```diff
@@ -120,7 +120,7 @@ get_dataframe_by_doi <- function(filedoi,
   filedoi <- prepend_doi(filedoi)
 
   # get_file can also take doi now
-  get_dataframe_by_id(file = filedoi, FUN = FUN, original = original, ...)
+  get_dataframe_by_id(fileid = filedoi, FUN = FUN, original = original, ...)
```

Comment:
I agree -- Thanks for catching that. 

As you can see, I changed the argument names so that `get_*_by_id`'s first argument is named `fileid`, `get_*_by_doi` is named `filedoi`, and `get_*_by_name` is named `filename`. I thought that would be make it clearer, but open to considering otherwise -- alternatively one could have set all the arguments to `file`.

Reply:
@wibeasley I fixed more of this inconsistency in 93f9c1